src/Resources/views/Navigation/navigation-blocks-BEM.html.twig

```diff
@@ -15,7 +15,7 @@
         {% set visibleNodes = root.children | filter(node => node.visible) %}
         {% if visibleNodes is not empty %}
             {% if level == 0  %}
-                <nav class="{{ navigation_base_class }} {% block navigation_additional_class %}{% endblock %}">
+                <nav aria-label="{{ navigation_landmark|default('Hauptnavigation') }}" class="{{ navigation_base_class }} {% block navigation_additional_class %}{% endblock %}">
```

Comment:
Ist das hier eine gute Idee? Würde man _dieses_ Partial via `extend`  o.ä. für alle möglichen Navigationen verwenden?

Ich frage mich, ob es hier nicht besser wäre, eine Exception zu werfen, wenn bei der Verwendung kein Name übergeben wird. Das trainiert uns, und es kann nicht passieren, dass Haupt-, Sub- und Footernavigation alle "Hauptnavigation" heißen.

Reply:
Good Point. D.h. hier einfach den Default Filter entfernen right?

Reply:
Das wäre allerdings ein BC.

Reply:
Hm, ja. Aber noch überschaubar und man könnte die 4-5 Projekte direkt fixen? Weiß nicht. Bei internen Paketen habe ich manchmal die Tendenz, es mit Semver nicht zu übertreiben, aber eine 5.0.0 kostet auch nix.

Leichte Tendenz zu 5.0.0 aber 🤷‍♀️ 

Auf >= 4.1 sind bisher:
<img width="370" alt="Bildschirmfoto 2022-03-24 um 14 56 47" src="https://user-images.githubusercontent.com/3763421/159932367-c96c20e8-a783-43f6-9274-f415a5b77a6e.png">

Reply:
Ich tendiere auch zu 5.x obwohl es so eine kleine Änderung ist.

Reply:
Alternativ könnten wir uns in der Ausgabe auch mit `{% if navigation_landmark is not defined %}{% verbatim %}Hast du nicht was vergessen?! {% endverbatim %}{% endif %}` anschreien.

Reply:
<img width="677" alt="Bildschirmfoto 2022-03-24 um 15 30 27" src="https://user-images.githubusercontent.com/7603184/159939155-7a69782c-8f82-4119-aa07-fac3e73bebc4.png">

Reply:
Ich tendiere dazu, mich anschreien zu lassen.

Reply:
Wo siehst du den Vorteil vom "visuellen Anschreien" gegenüber einer Exception? Ich habe das anfangs auch gern so gemacht, aber finde die Exception genauso aussagekräftig und noch, äh, lauter 😄

Reply:
Kein richtiger Vorteil. Es wäre halt nur keine major update sondern ein feature. Was für mich mehr der Natur der Änderung entspräche.

Reply:
Nunja, du könntest immerhin noch navigieren. :D

Reply:
Überzeugt. Ich mache eine 5er.